wrappers/ios/ci/build.sh

```diff
@@ -144,21 +157,24 @@ checkout_indy_sdk() {
 
 # NOTE: $INDY_SDK_DIR/libindy/target/$TRIPLET/release/libindy.a should be a non-fat file
 build_libindy() {
+    echo "ios/ci/build.sh: running build_libindy()"
     # OpenSSL-for-iPhone currently provides libs only for aarch64-apple-ios and x86_64-apple-ios, so we select only them.
     TRIPLETS="aarch64-apple-ios,x86_64-apple-ios"
 
     pushd $INDY_SDK_DIR/libindy
+        # TODO: we use lipo to build fat lib, but then we only used this libs in copy_libindy_architectures step
```

Comment:
This comment applies to the whole file:
Please move these TODO and question comments elsewhere.